xbmc/addons/PVRClient.cpp

```diff
@@ -1733,33 +1742,43 @@ xbmc_codec_t CPVRClient::cb_get_codec_by_name(const void* kodiInstance, const ch
   return CCodecIds::GetInstance().GetCodecByName(strCodecName);
 }
 
-CPVRClientCapabilities::CPVRClientCapabilities()
+CPVRClientCapabilities::CPVRClientCapabilities(const CPVRClientCapabilities& other)
 {
-  m_addonCapabilities = {0};
+  if (other.m_addonCapabilities.get())
+    m_addonCapabilities.reset(new PVR_ADDON_CAPABILITIES(*other.m_addonCapabilities.get()));
```

Comment:
Not a biggie, but you can omit the `.get()`:

```c++
m_addonCapabilities.reset(new PVR_ADDON_CAPABILITIES(*other.m_addonCapabilities));
```

Reply:
Will change that. Thanks.